tssc/step_implementers/tag_source/git.py

```diff
@@ -207,33 +209,51 @@ def _git_url(runtime_step_config):
             try:
                 return_val = sh.git.config(
                     '--get',
-                    'remote.origin.url').stdout.decode("utf-8").rstrip()
+                    'remote.origin.url',
+                    _out=sys.stdout,
+                    _tee=True,
+                    _encoding='UTF-8',
+                    _decode_errors='ignore'
+                    ).strip()
```

Comment:
I think I used rstrip() to default to a less aggressive strip.

Reply:
changed to rstrip